packages/flutter/lib/src/services/keyboard_maps.dart

```diff
@@ -842,3 +842,150 @@ const Map<int, PhysicalKeyboardKey> kFuchsiaToPhysicalKey = <int, PhysicalKeyboa
   0x000c028b: PhysicalKeyboardKey.mailForward,
   0x000c028c: PhysicalKeyboardKey.mailSend,
 };
+
+
+/// Maps macOS-specific USB HID Usage IDs to the matching [PhysicalKeyboardKey].
```

Comment:
These aren't USB HID values.  They are macOS keycode values, which represent physical keys on macOS.

The USB HID codes are standard codes defined here: https://www.usb.org/sites/default/files/documents/hut1_12v2.pdf